evap/staff/views.py

```diff
@@ -576,8 +576,18 @@ def semester_preparation_reminder(request, semester_id):
     responsible_list = [(responsible, [evaluation for evaluation in evaluations if responsible in evaluation.course.responsibles.all()],
                          responsible.delegates.all()) for responsible in responsibles]
 
-    template_data = dict(semester=semester, responsible_list=responsible_list)
-    return render(request, "staff_semester_preparation_reminder.html", template_data)
+    if request.method == "POST":
+        template = EmailTemplate.objects.get(name=EmailTemplate.EDITOR_REVIEW_REMINDER)
+        subject_params = {}
+        for responsible, evaluations, __ in responsible_list:
+            recipient = UserProfile.objects.get(pk=responsible.pk)
+            body_params = {"user": recipient, "evaluations": evaluations}
+            EmailTemplate.send_to_user(recipient, template, subject_params, body_params, use_cc=True, request=request)
+        messages.success(request, _("Successfully sent reminders to everyone."))
+        return HttpResponse("")
```

Comment:
So far, we've always used
```
return HttpResponse()  # 200 OK
```
Again, your code is fine, but I'd like to keep it consistent.